# SNV-analyzer
Custom R code to analyze sam files that were mapped to a reference sequence and converted using Calmd.

Analysis of Fastq data filesusing Galaxy (https://usegalaxy.eu). The quality of each sample of one NGS run should be reviewed using FastQC. Forward reads can be trimmed using Trimmomatic. The HEADCROP Trimmomatic operation was selected with 10 bases that were removed from the start of the read. The trimmed forward read and the reverse read can be merged using FLASH with a minimum overlap of 10, a maximum overlap of 200 and a maximum mismatch density of 0.25. In case the same index was used for different targets, Cutadapt can be performed to split the samples based on custom target adapter sequences. Matches were left unchanged, and a separate file was created for each adapter. Mapping was performed using “Map with BWA-MEM for medium and long reads ≥100 bp” using a 260 nt long reference sequence corresponding to the FGFR wildtype sequence in the respective region. The decision for the best algorithm was set to automatic and default settings were used, only for scoring options 0 was used as a penalty for a mismatch. Samtools calmd was used to change identical bases to “=” and the resulting BAM files were formatted to SAM files using Samtools view. Final SAM files were downloaded from Galaxy and used for analysis in R using this code.
